notebook/static/notebook/js/searchandreplace.js

```diff
@@ -188,20 +187,19 @@ define(function(require){
           $('<div/>').addClass('input-group')
           .append(
             $('<div/>').addClass('input-group-btn')
-              .append(isCaseSensitiveButton)
-              .append(isRegExpButton)
+                .append(isCaseSensitiveButton)
+                .append(isRegExpButton)
+                .append(onlySelectedButton)
           )
           .append(search)
         )
       )
       .append($('<div/>').addClass('form-group')
         .append(
-          $('<div/>').addClass('input-group')
-          .append(repl)
-          .append(
-            $('<div/>').addClass('input-group-btn')
-              .append(onlySelectedButton)
-          )
+          $('<div/>')
+              .addClass('input-group')
```

Comment:
why not just append the repl, rather than appending the input group div, and then putting repl inside that?